content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -243,11 +247,21 @@ metadata:
 maxReplicas: 100
 ```
 
+For each admission request, CEL expressions will be evaluated for each 
+(policy, binding, param) combination that match the request. For a request
+to be admitted it must pass ALL evaluations.
+
+If multiple bindings match the request, the policy will be evaluated for each,
+and they must all pass evaluation for the policy to be considered passed. 
+
+If multiple parameters match a single binding, the policy rules will be evaluated
+for each param, and they too must all pass for the binding to be considered passed.
+
 Bindings can have overlapping match criteria. The policy is evaluated for each matching binding.
 In the above example, the "nontest" policy binding could instead have been defined as a global policy:
 
 ```yaml
-apiVersion: admissionregistration.k8s.io/v1alpha1
+apiVersion: admissionregistration.k8s.io/v1beta1
```

Comment:
Could this use a [`codenew`](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files) shortcode?